internal/driver/glfw/menu_darwin.m

```diff
@@ -109,10 +109,18 @@ void handleException(const char* m, id e) {
         }
     }
 
-    if (index > -1) {
-        [menu insertItem:item atIndex:index];
+    if (strcmp(label, "About") == 0) {
```

Comment:
This branch looks like it should be done _before_ initializing a then unused item (which is also not released here).